src/ruby/puppet-server-lib/puppet/server/oids.rb

```diff
@@ -0,0 +1,31 @@
+module Puppet::Server::Oids
```

Comment:
As I mentioned below, it would probably be a good idea to add a `require 'yaml'` here.